### DISCUSSION TITLE
Save as new file, when in editor

### DISCUSSION THREAD

**Discussion by aisbergde** [gsantner/markor#2306] [Ideas]
Sometimes I want to split a file and I miss an easy way to quickly save the currently open file as a new file (save as), which is a very common file operation in nearly all editors.

also a "locate current file in Markor file explorer" could be helpful. In this case a copy operation could be done there.

**Comment by harshad1** [comment]
Share as plain text back into markor. Can then create a new file with the text.

**Reply by aisbergde** [reply]
Creating a copy by sharing is a complicated process. Markor is somewhere in the list of sharing targets, not on the first page. And then you need to check several pages of possible sharing targets until you find Markor. If it would be possible to force Markor to be on the first page of sharing targets, it would be OK. But maybe this sharing is controlled by the OS. 

Sharing in General is a hard process (I am using MIUI), because you never know, when and where to find a sharing target. On the first pages you get some targets in changing order, maybe depending on historical usage of apps. Later you get an alphabetical list.

Also sharing as file is too complicated. Because it's also based on the terrible sharing experience.

That's why I suggest this feature to make this more intuitive and with fewer steps, not depending on sharing.
